src/gui/EQ/EQPlot.h

```diff
@@ -7,6 +7,10 @@
 
 namespace gui::eq
 {
+
+void drawDbText (juce::Graphics& g, const chowdsp::SpectrumPlotBase& plot, float maxGain);
+void drawFrequencyText (juce::Graphics& g, const chowdsp::SpectrumPlotBase& plot, float frequency, float minGain);
```

Comment:
If we don't need these methods in the EqDrawView, then maybe these methods could like only in EQPlot.cpp? On the other hand, if we want to use these methods for some of the other spectrum plots, then maybe it would make more sense to put these methods in `gui/Shared/FrequencyPlotHelpers.h`?